CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## [Unreleased]
+[//]: # '## [Unreleased]'
```

Comment:
I thought we generally did the following for all our readmes. At least that's what I've noticed. 

```
<!--## [Unreleased] -->
```

Reply:
🤷‍♂️ I was just reverting to what it was before.

I definitely prefer the `<!--` syntax to the archaic `[//]: #`.